### PR TITLE
:art: Finish iteration 5.2

### PR DESCRIPTION
**Pull request by LeahWinters**
# Pull Request Template

## Description

When a previously saved outfit is on the bear:
     - a user can edit the outfit,
     - click the "Save" button,
     - and see that the outfit is permanently updated,
     - including in `localStorage`

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

gui tested

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -72,20 +72,37 @@ function displayChoice(choiceType) {
 }
 
 function saveOutfit() {
-  if (currentOutfit.title === undefined) {
-    currentOutfit.setTitle(outfitNameInput.value);
+  debugger;
```
Comment (J-Poulter) [inline main.js]: You left a debugger in the code,  can you please remove this before the merge

File: main.js [hunk 2]
```diff
@@ -72,20 +72,37 @@ function displayChoice(choiceType) {
...
+function saveNewOutfit() {
   populateCard();
   var newOutfit = new Outfit(currentOutfit);
   outfitList.push(newOutfit);
-  saveToLocalStorage(newOutfit);
-  resetHelper();
+}
+
+function updateExistingOutfit() {
+  // debugger;
```
Comment (J-Poulter) [inline main.js]: Here's another debugger leftover in the code, can you remove it as well please.

**Comment by J-Poulter** [comment]
Everything looks great, after we remove the leftover debuggers I think it's good to merge.  Then we can begin on refactoring code where we think it needs it.

File: main.js (hunk 1 above)
Comment (LeahWinters) [reply main.js]: I'll go in and fix it!

File: main.js (hunk 2 above)
Comment (LeahWinters) [reply main.js]: Sweet! Thanks for catching that!

**Pull request merged**